eth2util/keystore/keystore.go

```diff
@@ -411,3 +394,24 @@ func randomHex32() (string, error) {
 
 	return hex.EncodeToString(b), nil
 }
+
+// validateKeystoreFilename validates keystore filename and returns index of the keystore file.
+func validateKeystoreFilename(file string) (int, error) {
```

Comment:
What does `validate` means in this context?

Reply:
not sure what you mean by the context, IMO if I read `validateKeystoreFilename` it is clear that it checks if the given keystore filename is valid or not.

Can modify godoc to:
// validateKeystoreFilename checks if the given keystore filename is valid. and returns index of the keystore file.

Reply:
validate shouldn't return data, suggest:`getKeystoreIndex`

Reply:
> not sure what you mean by the context

I meant to say to explain in the comment doc when `file`  is valid, and when it isn't.